src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upSide;
+    private final double downSide;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double upSide, double downSide, double height) {
+        super(color);
+        this.upSide = upSide;
+        this.downSide = downSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (upSide + downSide) * height / 2;
+    }
+
+    @Override
+    public String toString() {
+        System.out.println("IsoscelesTrapezoid{" + "upSide=" + upSide
+                + ", downSide=" + downSide
+                + ", height=" + height + ", color='"
+                + getColor() + '\'' + ", area=" + getArea() + '}');
```

Comment:
behaviour - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() `(you shouldn't override toString() method for this).`